src/System.Net.Primitives/src/System/Net/CredentialCache.cs

```diff
@@ -144,13 +145,6 @@ public void Remove(string host, int port, string authenticationType)
             _cacheForHosts.Remove(key);
         }
 
-        /// <devdoc>
```

Comment:
Why the requirement to keep these? I agree with keeping useful devdocs, but these in particular seem pretty low value to me, bordering on noise (e.g. the ctor comment says the obvious: "Initializes a new instance", the comment for `Add` says the obvious: "Adds a NetworkCredential", the comment for `Remove` says the obvious: "Removes a NetworkCredential", etc.), and as you mention, these devdocs aren't applied consistently throughout this file.